README.md

```diff
@@ -81,16 +40,62 @@ Expected output:
 }
 ```
 
-## Configuration
+## Installation
+### Docker
+1. Install Docker
+1. Set up a directory to contain your own JKS file.
+1. Set up a subdirectory in the above directory called "caCerts" to contain any CA certificates (one per file) from the remote system that need to be installed in the Java truststore on boot.
+1. Copy the sample.env file to a new '.env' file and populate with your settings. See [Configuration](#configuration) for more information.
+1. Build & run the application using `docker compose up --build -d`
+1. To spin down the container, run `docker compose down`
+
+### Manual
+1. Install Maven
+1. Install Java
+1. Configure the properties file as needed (see [Configuration](#configuration))
+1. Package the application using `mvn clean install`
+1. Run the application using `java -jar target/jpo-security-svcs-0.0.1-SNAPSHOT.jar`
 
+## Configuration
 In `./src/main/resources/application.properties` you'll find the following properties which can be defined whether on the command line or by environment variable. To define the property on the command line, insert `--` to the front of the Property name, for example, `--server.port=8091`:
 
 | Property | Meaning | Default Value | Environment Variable Substitute |
-| -----------|------------|-----------------|-----------|
-| server.port | The port number to which this service will be listening.| 8090 |SERVER_PORT|
-| sec.useHsm | Whether to use an HSM or not. | false | SEC_USE_HSM |
-| sec.cryptoServiceBaseUri | Cryptographic service endpoint URI excluding path. For example, `http://<ip>:<port>` OR `http://server.dns.name` including the port number, if any. | - |SEC_CRYPTO_SERVICE_BASE_URI|
-| sec.cryptoServiceEndpointSignPath | The REST endpoint path of the external service. | /tmc/signtim |SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH|
+| -------- | ------- | ------------- | ------------------------------- |
+| server.port | The port number to which this service will be listening. | 8090 | SERVER_PORT |
+| sec.cryptoServiceBaseUri | Cryptographic service endpoint URI excluding path. For example, `http://<ip>:<port>` OR `http://server.dns.name` including the port number, if any. | - | SEC_CRYPTO_SERVICE_BASE_URI|
+| sec.cryptoServiceEndpointSignPath | The REST endpoint path of the external service. | /tmc/signtim |SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH |
 | sec.useCertificates | Whether to use certificates or not. | true | SEC_USE_CERTIFICATES |
 | sec.keyStorePath | The path to the keystore file. | /home/cert.jks | SEC_KEY_STORE_PATH |
 | sec.keyStorePassword | The password for the keystore file. | password | SEC_KEY_STORE_PASSWORD |
+
+## Mutual TLS Authentication
+For enhanced security, MTLS is used to communicate with a remote signing system. This requires properly configured certificates from the remote system to property perform - work with your remote signing authority to obtain necessary certificates. 
```

Comment:
Should this be 'properly perform' instead of 'property perform'?

Reply:
Yes, thanks for catching that! This has been fixed!